ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -200,12 +263,15 @@ module Ldap = struct
       ; password_expired= logand user_account_control passw_expire_bit <> 0l
       }
 
-  let query_user sid domain_netbios kdc =
+  let env_of_krb5 domain_netbios =
     let domain_krb5_cfg =
       Filename.concat domain_krb5_dir
         (Printf.sprintf "krb5.conf.%s" domain_netbios)
```

Comment:
Is this created by winbind itself? I can't find any code in xapi that would create this file

Reply:
yes, winbind create it, we just re-use it (for the KDCs configured)